app/src/bin/crate.in.sh

```diff
@@ -46,11 +46,19 @@ if [ "x$CRATE_USE_IPV4" != "x" ]; then
   JAVA_OPTS="$JAVA_OPTS -Djava.net.preferIPv4Stack=true"
 fi
 
+## GC configuration
 JAVA_OPTS="$JAVA_OPTS -XX:+UseConcMarkSweepGC"
-
 JAVA_OPTS="$JAVA_OPTS -XX:CMSInitiatingOccupancyFraction=75"
 JAVA_OPTS="$JAVA_OPTS -XX:+UseCMSInitiatingOccupancyOnly"
 
+## G1GC Configuration
+# NOTE: G1GC is only supported on JDK version 10 or later.
+# To use G1GC uncomment the lines below.
```

Comment:
I don't think we're advocating to edit this file, do we?

Most packages afaik would overwrite it without warning/diff file on a package upgrade.